src/nvim/ui.h

```diff
@@ -53,7 +53,12 @@ struct ui_t {
   bool ui_ext[kUIExtCount];  ///< Externalized UI capabilities.
   int width;
   int height;
+  int pum_nlines; /// actual nr. lines shown in PUM
```

Comment:
`ui_pum_get_height` needs to be changed to use this.

Reply:
oops. 🆗